install fails on /etc/bash_completion.d/mscs

I recently tried installing mscs on an Ubuntu 22 server. I got this error:
```
install -m 0755 msctl /usr/local/bin/msctl
install -m 0755 mscs /usr/local/bin/mscs
install -m 0644 mscs.completion /etc/bash_completion.d/mscs
install: cannot create regular file '/etc/bash_completion.d/mscs': No such file or directory
make: *** [Makefile:34: update] Error 1
```
I investigated and found that my server has no `/etc/bash_completion.d` directory even though the `bash-completion` apt package is installed.
I found [this stackexchange discussion](https://unix.stackexchange.com/questions/416185/where-to-install-bash-completion-scripts-for-out-of-tree-packages) saying that `/etc/bash_completion.d/` is a legacy dir and bash completions should instead go into `/usr/share/bash-completion/completions/`.

I also looked in `/usr/share/bash-completion/completions/` on my barebones Ubuntu 22 server and found it full of completion files, so it does seems like this is where the completion file should live.

I wonder though, is this something unique to Ubuntu? or do modern Debian releases and other linux distros also use `/usr/share/bash-completion/completions/`?

--- a/Makefile
+++ b/Makefile
@@ -6,7 +6,16 @@ MSCS := /usr/local/bin/mscs
 MSCS_INIT_D := /etc/init.d/mscs
 MSCS_SERVICE := /etc/systemd/system/mscs.service
 MSCS_SERVICE_TEMPLATE := /etc/systemd/system/mscs@.service
-MSCS_COMPLETION := /etc/bash_completion.d/mscs
+
+# Bash completion: detect proper location
+BASH_COMPLETION_DIR := $(shell \
+	if [ -d /usr/share/bash-completion/completions ]; then \
+		echo /usr/share/bash-completion/completions; \
+	else \
+		echo /etc/bash_completion.d; \
+	fi)
+
+MSCS_COMPLETION := $(BASH_COMPLETION_DIR)/mscs
 
 UPDATE_D := $(wildcard update.d/*)
 
@@ -21,12 +30,12 @@ install: adduser update
 	fi
 	
 adduser:
-        # safety check to see if user exists before trying to create it 
+	# safety check to see if user exists before trying to create it 
 	if id $(MSCS_USER); then \
-                echo "Minecraft user $(MSCS_USER) exists so not creating it"; \
-        else \
-                useradd --system --user-group --create-home -K UMASK=0022 --home $(MSCS_HOME) $(MSCS_USER); \
-        fi
+		echo "Minecraft user $(MSCS_USER) exists so not creating it"; \
+	else \
+		useradd --system --user-group --create-home -K UMASK=0022 --home $(MSCS_HOME) $(MSCS_USER); \
+	fi
 
 update:
 	install -m 0755 msctl $(MSCTL)

